sdk/aqueduct/backend/response_models.py

```diff
@@ -249,3 +249,9 @@ class GetVersionResponse(BaseModel):
     """This is the response object returned by /api/version."""
 
     version: str
+
+
+class EngineStatusResponse(BaseModel):
```

Comment:
Since it's just tracking K8s cluster status maybe we can just call it `K8sEngineStatusResponse`?

Reply:
Maybe I'll call it `DynamicEngineStatusResponse`.